### PR TITLE
navigation bar tool

### PR DESCRIPTION
**Pull request by Agaba-derrick**

### CODE REVIEW

**Comment by Agaba-derrick** [comment]
please accept this

**Comment by jnsereko** [comment]
please  @Agabaderrickjunior 

1. squash your commits   https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits
2. add issue link 
3. add a short video / screen capture of your changes 

Thank for the work.

File: .github/supportus.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# These are supported funding model platforms
+
+github: # Replace with up to 4 GitHub Sponsors-enabled usernames e.g., [user1, user2]
+patreon: # Replace with a single Patreon username
```
Comment (jnsereko) [inline .github/supportus.yml:4]: does this also satisfy https://github.com/richarddushime/gdsc/issues/7

File: style.css [hunk 2]
```diff
@@ -63,3 +63,21 @@ p{
...
+    list-style-type: none;
+    margin:0;
+    padding: 0;
+    overflow: 0;
+    background-color: #31A6FF;
+}
+li {
+    float: left;
+}
+li a{
```
Comment (jnsereko) [inline style.css:76]: which element is this targeting?

File: new changes [hunk 3]
```diff
@@ -4,7 +4,13 @@
...
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <a href="https://bugemauniv.ac.ug/index.php/index.php?schoolofcomputingandinformatics">Bugema Universty school Of IT</a>
+    <a href="https://bugemauniv.ac.ug/index.php/index.php?schoolofcomputingandinformatics">Bugema Universty School Of IT</a>
+    <ul>
+        <li><a href="HOME">HOME</a></li>
+        <li><a href="CONTACT US">CONTACT</a></li>
+        <li><a href="ABOUT US">ABOUT US</a> </li>
+        <li><a href="SIGN IN">SIGN IN</a></li>
+    </ul>
```
Comment (jnsereko) [inline new changes:13]:
```
<div class="nav">
    <ul>
        <li><a href="HOME">HOME</a></li>
        <li><a href="CONTACT US">CONTACT</a></li>
        <li><a href="ABOUT US">ABOUT US</a> </li>
        <li><a href="SIGN IN">SIGN IN</a></li>
    </ul>
</div>
```

**Review by jnsereko — CHANGES_REQUESTED** [review]

**Review by mherman22 — CHANGES_REQUESTED** [review]
Your pull request is not descriptive enough. A little explanation of what your pull request does

**Pull request merged**